test/new-e2e/scenarios/system-probe/main.go

```diff
@@ -52,6 +52,8 @@ func main() {
 	dependenciesDirectoryPtr := flag.String("dependencies-dir", DD_AGENT_TESTING_DIR, "directory where dependencies package is present")
 	vmconfigPathPtr := flag.String("vmconfig", defaultVMConfigPath, "vmconfig path")
 	local := flag.Bool("local", false, "is scenario running locally")
+	runAgentPtr := flag.Bool("run-agent", false, "Run datadog agent on the metal instance")
+	agentVersionPtr := flag.String("agent-version", "7.49", "Version of datadog-agent")
```

Comment:
For a future PR: we should explore having this dynamically set to the latest released version, so we don't have to keep updating it.

Reply:
FWIW: If you do not specify a version for the agent, the framework should automatically install the latest version of the agent. cf https://github.com/DataDog/test-infra-definitions/blob/main/components/datadog/agentparams/params.go#L56